jni_gen/lib/src/util/rename_conflict.dart

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) 2022, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+String renameConflict(Map<String, int> counts, String name) {
+  if (counts.containsKey(name)) {
+    final count = counts[name]!;
+    final renamed = '$name$count';
+    counts[name] = count + 1;
+    return renamed;
+  }
+  counts[name] = 1;
+  return kwRename(name);
+}
+
+// Concats 0 to name if name is a keyword
```

Comment:
keyword -> keyword.

Reply:
Maybe

/// If name is a keyword, postfixes 0.
///
/// Examples:
/// class -> class0
/// foo -> foo